flips/20211007-transaction-fees.md

```diff
@@ -138,93 +137,92 @@ Inclusion effort can represent many different parts of the system. Listed below
+To get the terms of this linear function, we need a way to quantify the impact of the transaction byte size on the network. This can be done by taking a reference transaction that is at the 95th percentile of transaction byte sizes currently seen on mainnet, and does very little computation. This transaction should have inclusion fees of half of the current static fees. We then see how many transactions like this the network can handle per second before it runs into problems. We can define this as the saturation point. The transaction's saturation point is inversely proportionate to the fees that should be charged for the transaction. If the network can handle half as many transactions $T_1$ as transactions $T_2$, then $T_1$ should be twice as expensive as $T_2$. Using this relation and getting the saturation points for a few transactions of different sizes, we can calibrate the linear dependency.
 
-The inclusion effort cost parameter is a free variable in the inclusion fees equation, so we can define the inclusion effort cost parameter such that the coefficient <img src="https://render.githubusercontent.com/render/math?math=k"> is 1/1000. This means the Inclusion fees can be interpreted as some base fee, plus 1 <img src="https://render.githubusercontent.com/render/math?math=p_I"> for each kilobyte.
+The inclusion effort cost parameter is a free variable in the inclusion fees equation, so we can define the inclusion effort cost parameter such that the coefficient $k$ is 1/1000. This means the inclusion fees can be interpreted as some base fee, plus 1 $p_I$ for each kilobyte.
 
 #### Execution fees
 
-Execution effort can be kept as the execution effort measuring that is currently in place. This can be later upgraded so the execution effort, more closely matches the resource usage of the transaction.
+The execution effort can be kept as the execution effort measuring that is currently in place. This can be later upgraded so the execution effort more closely matches the resource usage of the transaction.
```

Comment:
Although, I do understand the meaning, this sentence sounds a bit "tautological" and may need to be rewritten.